.github/workflows/cd-syft.yml

```diff
@@ -137,6 +137,36 @@ jobs:
             echo "github_release_version=$(python packages/grid/VERSION)" >> $GITHUB_OUTPUT
           fi
 
+      # Checkout to gh-pages and update helm repo
+      - name: Checkout to gh-pages
+        uses: actions/checkout@v3
+        with:
+          ref: gh-pages
+          token: ${{ secrets.SYFT_BOT_COMMIT_TOKEN }}
+
+      - name: Copy helm repo files from Syft Repo
+        run: |
+          git checkout dev -- packages/grid/helm/repo/
+          git reset gh-pages
+          cp -R packages/grid/helm/repo/. helm/
+          rm -rf packages/
+
+      - name: Commit changes to gh-pages
+        uses: EndBug/add-and-commit@v9
+        with:
+          author_name: ${{ secrets.OM_BOT_NAME }}
+          author_email: ${{ secrets.OM_BOT_EMAIL }}
+          message: "Update Helm package from Syft Repo"
+          add: "helm/"
+          push: "origin gh-pages"
+
+      # Checkout dev
+      - name: Checkout to dev Branch
```

Comment:
MAYBE: we could remove this, if we are moving to last the helm gh-pages upload, as it would not be used

Reply:
Resolved by: 6b3c689baf8973f5021b1217f5557d58c362d02d